tests/multi_asic_intfutil_test.py

```diff
@@ -0,0 +1,150 @@
+import os
+import subprocess
+from unittest.case import TestCase
```

Comment:
better to use pytest instead of unittest here.

Reply:
Fixed in the latest commit